src/HomeKit/HMMutablePresenceEvent.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.HomeKit {
+
+	partial class HMMutablePresenceEvent {
+
+		public HMPresenceType PresenceType {
```

Comment:
Because BindAs will generate a virtual member so making it virtual will be a breaking change, also we will need to use the same property name to it is also not a breaking change